tests/test_sft_trainer.py

```diff
@@ -808,3 +809,62 @@ def test_pretokenized_dataset_wrong_format():
         # is essentially swallowing a KeyError here.
         with pytest.raises(ValueError):
             sft_trainer.train(MODEL_ARGS, data_args, train_args, PEFT_PT_ARGS)
+
+
+def test_set_log_level_for_logger_default():
+    """
+    Ensure that the correct log level is being set
+    for python native logger and transformers logger
+    """
+
+    # Set env var TRANSFORMERS_VERBOSITY as None and test
+    os.unsetenv("TRANSFORMERS_VERBOSITY")
+    os.unsetenv("LOG_LEVEL")
+    os.unsetenv("SFT_TRAINER_CONFIG_JSON_ENV_VAR")
+    os.unsetenv("SFT_TRAINER_CONFIG_JSON_PATH")
```

Comment:
Another way you can do this is by mocking the env:
```py
# decorator can be used on whole function
@mock.patch.dict(os.environ, {}, clear=True)

# or can create context scope
with mock.patch.dict(os.environ, {}, clear=True):
```

Reply:
Thank you!